lxd/instance/instance_interface.go

```diff
@@ -95,6 +96,7 @@ type Instance interface {
 	VolatileSet(changes map[string]string) error
 
 	// File handling.
+	FileSocket() (net.Conn, error)
```

Comment:
Can this be called FileSFTPConn (or Socket) to indicate the connection/socket returned is for an SFTP service.